stack/base-with-services/Dockerfile

```diff
@@ -29,36 +30,34 @@ ENV PGDATA=/home/${NB_USER}/.postgresql
 RUN if [ "$TARGETARCH" = "amd64" ]; then \
      mamba create -p /opt/conda/envs/aiida-core-services --yes \
          postgresql=${PGSQL_VERSION} \
-         rabbitmq-server=3.8.14  && \
+         rabbitmq-server=${RMQ_VERSION} && \
      mamba clean --all -f -y && \
-     fix-permissions "${CONDA_DIR}" && \
-     fix-permissions "/home/${NB_USER}"; \
-elif [ "$TARGETARCH" = "arm64" ]; then \
+     fix-permissions "${CONDA_DIR}"; \
+  elif [ "$TARGETARCH" = "arm64" ]; then \
      mamba create -p /opt/conda/envs/aiida-core-services --yes \
          postgresql=${PGSQL_VERSION} && \
      mamba clean --all -f -y && \
-     export RMQ_VERSION=3.9.13 && \
      wget -c https://github.com/rabbitmq/rabbitmq-server/releases/download/v${RMQ_VERSION}/rabbitmq-server-generic-unix-${RMQ_VERSION}.tar.xz && \
      tar -xf rabbitmq-server-generic-unix-${RMQ_VERSION}.tar.xz && \
      rm rabbitmq-server-generic-unix-${RMQ_VERSION}.tar.xz && \
-     mv rabbitmq_server-${RMQ_VERSION} /opt/conda/envs/aiida-core-services/ && \
-     fix-permissions "${CONDA_DIR}" && \
-     fix-permissions "/home/${NB_USER}"; \
-     ln -sf /opt/conda/envs/aiida-core-services/rabbitmq_server-${RMQ_VERSION}/sbin/* /opt/conda/envs/aiida-core-services/bin/; \
-fi
+     mv rabbitmq_server-${RMQ_VERSION} /opt/conda/envs/aiida-core-services/rabbitmq_server && \
+     ln -sf /opt/conda/envs/aiida-core-services/rabbitmq_server/sbin/* /opt/conda/envs/aiida-core-services/bin/ && \
+     fix-permissions "${CONDA_DIR}"; \
+  fi
 
 # Configure AiiDA profile.
 COPY config-quick-setup.yaml .
 COPY before-notebook.d/20_start-postgresql.sh /usr/local/bin/before-notebook.d/
-COPY before-notebook.d/30_start-rabbitmq-${TARGETARCH}.sh /usr/local/bin/before-notebook.d/
+COPY before-notebook.d/30_start-rabbitmq.sh /usr/local/bin/before-notebook.d/
+
+# Configure RabbitMQ
+COPY configure_rabbitmq.sh .
```

Comment:
NOTE: I've separated part of the startup script related to RMQ configuration and now it runs during image build and not at container startup. This makes startup a tiny bit faster, but the main benefit for me that the rmq startup script (`30_start-rabbitmq.sh`` is now very simple.